### ISSUE TITLE
Benchmark Lambda with 10,240 MB

### ISSUE THREAD

**Issue by andreaswittig** [sjakthol/aws-network-benchmark#1]
@sjakthol Great work. Is there a reason why you did not benchmark a Lambda function with the maximum of 10,240 MB memory?

**Comment by lovio** [comment]
I have tested using an real-world S3 workload with the AWS SDK. The baseline and burst performance are the same with the results using iperf3, and there are no differences between the various memory(vCPU) configurations.

**Comment by sjakthol** [comment]
I initially did this test four years ago and then Lambda maximum memory was 3008 MB. I did do a rerun to get fresh numbers this year. But I did not change or add new tests while doing that.

As @lovio commented, there's really no variance in Lambda network throughput with different memory (vCPU) configurations. The numbers from this test also support this as pretty much all the sizes that were tested had the same network throughput.

I did a small, separate test with 10 GB memory now. Based on that it seems that you get the same throughput with 10 GB memory as you get with all other tested memory values.

I've added a note about this separate 10 GB test and its results to the README: https://github.com/sjakthol/aws-network-benchmark/commit/b355d4fc249b09414e265dbc3a66a66c4589cd4e#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5.
